contrib/ci/configure.sh

```diff
@@ -29,62 +29,43 @@ declare -a CONFIGURE_ARG_LIST=(
     "--enable-ldb-version-check"
     "--with-syslog=journald"
     "--enable-systemtap"
-    "--with-python2-bindings"
 )
 
-
-if [[ "$DISTRO_BRANCH" == -redhat-redhatenterprise*-6.*- ||
-      "$DISTRO_BRANCH" == -redhat-centos-6.*- ]]; then
+if [[ "$DISTRO_BRANCH" == -redhat-centos-8*- ||
+      "$DISTRO_BRANCH" == -redhat-redhatenterprise*-8.*- || ]]; then
     CONFIGURE_ARG_LIST+=(
-        "--with-smb-idmap-interface-version=5"
-        "--disable-cifs-idmap-plugin"
-        "--with-syslog=syslog"
-        "--without-python3-bindings"
-        "--without-kcm"
+        "--with-python2-bindings"
     )
-fi
-
-if [[ "$DISTRO_BRANCH" == -redhat-fedora-2[0-2]* ]]; then
+else
     CONFIGURE_ARG_LIST+=(
-        "--without-kcm"
+        "--without-python2-bindings"
```

Comment:
Btw:
```
--with-python2-bindings Whether to build python2 bindings [no]
```
  --  so I realized this `else` can be actually omitted.

But I'm fine with current version as well.

Reply:
I'd prefer to state it explicitly